public/static/locales/en/form-builder.json

```diff
@@ -699,5 +699,22 @@
     "description": "Download a copy of the form and try again or",
     "supportLink": "contact Support",
     "errorCode": "Error code: {{code}}"
+  },
+  "addConditionalRules": {
+    "modalTitle": "Add a rule to a question",
+    "modalTitleEdit": "Edit rules",
+    "modalDescription": "Allow an option in your question to show an additional short or long answer input field when selected. Only drop downs, single choice, and multi-select blocks can apply a rule.",
+    "optionTitle": "If this option is selected:",
+    "questionTitle": "Show question:",
+    "text1": "Show when: ",
+    "text2": "Has value:",
+    "defaultQuestion": "No conditions always show",
```

Comment:
@timarney Having a hard time finding where these strings show up while clicking around in the preview environment? Are they experimental for input field value conditionals?

Reply:
Might have been removed I'll double check in the A.M